src/graphql/helpers.ts

```diff
@@ -31,7 +31,15 @@ export function formatAddress(address: string): string {
   try {
     return getAddress(address);
   } catch (error) {
-    // return same address for now, but return error in the future, if address is not EVM or sn address
+    // Pad starknet address to 64 characters
```

Comment:
Can we support Starknet address without going through `getAddress` function before?

Reply:
Yes, I plan to move this function to the SDK, so we can reuse it inside sequencer and hub, with a more robust address detection